### PR TITLE
 Scrolling Tabs in TabControl, Better horizontal scrolling 

### PR DESCRIPTION
**Pull request by M0n7y5**
vLGlhvvB1e

### CODE REVIEW

File: src/Shared/HandyControl_Shared/Controls/Other/ScrollViewer.cs [hunk 1]
```diff
@@ -17,6 +17,8 @@ public class ScrollViewer : System.Windows.Controls.ScrollViewer
 
         private bool _isRunning;
 
+        private Storyboard _scrollingAnimation = new Storyboard();
```
Comment (NaBian) [inline src/Shared/HandyControl_Shared/Controls/Other/ScrollViewer.cs:20]: @M0n7y5 emmm, can you tell me the use of _scrollingAnimation? Or why commented BeginAnimation on line 123 but not on line 141?

Comment (M0n7y5) [reply src/Shared/HandyControl_Shared/Controls/Other/ScrollViewer.cs:20]: Oh ... That's a my mistake ... Sorry a forgot to commit my recent code, so this pull request contains old mess 👀 Can i just close this pull request and make a new ?

Comment (NaBian) [reply src/Shared/HandyControl_Shared/Controls/Other/ScrollViewer.cs:20]: Just let me do it.

Comment (M0n7y5) [reply src/Shared/HandyControl_Shared/Controls/Other/ScrollViewer.cs:20]: k

**Pull request merged**